documentation/asciidoc/accessories/display/display_intro.adoc

```diff
@@ -1,47 +1,73 @@
+
+Next, connect the other end of the black wire to the GND pin on the display and the other end of the red wire to the 5V pin on the display. Once all the connections are made, you should see the Touch Display turn on the next time you turn on your Raspberry Pi.
+
+The other three pins on the Touch Display are used to connect the display to an original Raspberry Pi 1 Model A or B. Refer to our documentation on xref:display.adoc#legacy-support[legacy support] for more information.
+
+NOTE: An original Raspberry Pi is easily identified compared to other models; it is the only model with a 26-pin rather than 40-pin GPIO header connector.
+
+==== Power the Touch Display directly
+
+If you don't want to use a Raspberry Pi to provide power to the Touch Display, you can use a micro USB power supply instead. We recommend using the https://www.raspberrypi.com/products/micro-usb-power-supply/[Raspberry Pi 12.5W power supply] to make sure the display runs as intended.
+
+Do not connect the GPIO pins on your Raspberry Pi to the display if you choose to use micro USB for power. The only connection between the two boards should be the ribbon cable.
+
+WARNING: If you use a micro USB cable to power the display it must be mounted inside a chassis to reduce the risk of electrocution. The chassis also must block access to the display's PCB while it's in use.
 
 === Screen Orientation
 
-LCD displays have an optimum viewing angle, and depending on how the screen is mounted it may be necessary to change the orientation of the display to give the best results. By default, the Raspberry Pi Touch Display and Raspberry Pi are set up to work best when viewed from slightly above, for example on a desktop. If viewing from below, you can physically rotate the display, and then tell the system software to compensate by running the screen upside down.
+LCD displays have an ideal viewing angle, and it may be necessary to change the orientation of the display to find the perfect angle. If you need to physically rotate the display, or mount it in a specific position, you can use software to adjust the orientation of the screen to better match your setup.
 
-To set screen orientation when running the graphical desktop, select the `Screen Configuration` option from the `Preferences` menu. Right click on the DSI display rectangle in the layout editor, select Orientation then the required option.
+To set the screen orientation from the desktop environment, select the **Screen Configuration** option from the **Preferences** menu. Right click on the DSI display rectangle in the layout editor, select **Orientation**, then pick the best option to fit your needs.
 
-To set screen orientation when in console mode, you will need to edit the kernel command line to pass the required orientation to the system.
+To set screen orientation while in console mode, you need to edit the kernel's `cmdline.txt` file to pass the required orientation to the system.
```

Comment:
"console mode" -> "from the command line"

Where is the `cmdline.txt` file located?

Reply:
Changed to command line, and the file's location is shown in the source block underneath the paragraph.

Reply:
I think the reason I originally used "console mode" rather than "from the command line" is because people might think that running a Terminal in the desktop environment also counts as "from the command line"? I guess you could use "When booting to the command line" as IIRC that matches the language used in `Raspberry Pi Configuration` / `raspi-config` ?